pubspec.yaml

```diff
@@ -17,6 +17,12 @@ dependencies:
   latlong: ^0.6.1
   flutter_map_marker_popup: ^0.2.0
 
+dependency_overrides:
```

Comment:
why dependency_overrides?

Reply:
As I told

>We must wait for rorystephenson/flutter_map_marker_popup#17 to be merged to remove dependency_overrides

As I wan too busy to fix it completely it's not merged but thanks to @rorystephenson now you can remove dependency_overrides and use `flutter_map_marker_popup: ^0.3.0`

Reply:
pls fix it and reopen

Reply:
@lpongetti I tried to upgrade to `flutter_map_marker_popup: ^0.3.0` but unfortunately `MarkerPopup` was removed and currently I'm too busy to fix it